e2e/manifests/values/deployment-scheduler.yaml

```diff
@@ -32,21 +32,17 @@ scheduler:
 lvmd:
   managed: false
 
+storageClasses:
```

Comment:
It looks like you moved it by mistake.. Would you revert this if unnecessary?

Reply:
It was indeed a mistake, sorry about that. Fixing.